src/components/TimePicker/TimePicker.tsx

```diff
@@ -45,14 +41,14 @@ const NUM_PAD_VIEW_ID = 'numPadView';
 
 /**
  * Replace the sub-string of the given string with the provided value
- * @param {String} originalString - the string that will be modified
- * @param {String} newSubstring - the replacement string
- * @param {Number} from - the start index of the sub-string to replace
- * @param {Number} to - the end index of the sub-string to replace
+ * @param originalString - the string that will be modified
+ * @param newSubstring - the replacement string
+ * @param from - the start index of the sub-string to replace
+ * @param to - the end index of the sub-string to replace
  *
- * @returns {String} - the modified string with the range (from, to) replaced with the provided value
+ * @returns - the modified string with the range (from, to) replaced with the provided value
  */
-function insertAtPosition(originalString, newSubstring, from, to) {
+function insertAtPosition(originalString: string, newSubstring: string, from: number, to: number) {
```

Comment:
Add return type if it's other than void (for other functions too)

Reply:
Shouldn't we rely on type inference whenever possible?

Reply:
I'm fine with inference but not all people are and the standard I got used to is to add return types other than void 🤷‍♂️

Reply:
It's not perfect, but it works so far